.travis.yml

```diff
@@ -1,16 +1,14 @@
 language: c
-sudo: required
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
-script: bash -ex .travis-opam.sh
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docker.sh
+script: bash -ex .travis-docker.sh
+services:
+- docker
 env:
   global:
-  - PACKAGE=calendar
+  - PINS="calendar:."
+  - DISTRO="ubuntu-16.04"
   matrix:
-  - OCAML_VERSION=4.03
-  - OCAML_VERSION=4.04
-  - OCAML_VERSION=4.05
-  - OCAML_VERSION=4.06
-  - OCAML_VERSION=4.07
-os:
-  - linux
-  - osx
+  - PACKAGE="calendar" OCAML_VERSION="4.03.0" TESTS=1 EXTRA_DEPS="alcotest"
```

Comment:
from what i remember the new docker containers for `opam2` do not use the minor version for ocaml switch names anymore. https://github.com/ocaml/infrastructure/wiki/Containers#selecting-a-specific-compiler